release-notes.txt

```diff
@@ -1,3 +1,354 @@
+#4053: macros: RIOT_FILE_RELATIVE printing wrong file name for headers
+
+
+Other issues:
+-------------
+#10800: iotlab-m3: thread tests failing
+#10751: Possible memset optimized out in crypto code
+#10731: nanocoap: incomplete response to /.well-known/core request
+#10639: sys/stdio_uart: dropped data when received at once
+#10510: xtimer_set_msg: crash when using same message for 2 timers
+#10468: Tinycryt upstream rewrote history in master branch
+#10463: stm32f4discovery isn't visible as ttyUSB after flashing with tests/leds
+#10345: frdm-k22f cannot flash after certain firmware flashed
+#10341: driver_my9221 prevents further flashing of nucleo-l073rz
+#10206: Lua_basic example doesn't flash in b-l072z-lrwan1
+#10175: No error returned from aes_init when a key with a bad size is used
+#10121: RIOT cannot compile with the latest version of macOS (10.14) and Xcode 10
+#9882: sys/tsrb is not thread safe on AVR
+#9518: periph/i2c: tracking bugs and untested acks
+#9371: assert: c99 static_assert macro doesn't function for multiple static_asserts in the same scope
+#8975: dist/tools/openocd: cannot debug some stlink based boards
+#8653: msba2: default example fails on assert
+#8589: Why using -F in avrdude?
+#8436: Kinetis PhyNode: failed to flash binary > 256K
+#8107: crypto/ccm: bugs in the implementation of CCM mode
+#7220: sys/fmt: Missing tests for fmt_float, fmt_lpad
+#7199: core: "Invalid read of size 4"
+#6874: SAMD21: possible CMSIS bug ?
+#6533: tests/lwip target board for python test is hardcoded to native
+#5561: C++11 extensions in header files
+#5218: some use of asm keyword might be missing volatile
+#5009: RIOT is saw-toothing in energy consumption (even when idling)
+#4866: periph: GPIO drivers are not thread safe
+#4512: pkg: tests: RELIC unittests fail on iotlab-m3
+#4488: Making the newlib thread-safe
+#3256: make: Setting constants on compile time doesn't really set them everywhere
+#2346: Tracker: Reduce scope on unintended COMMON variables
+#2175: ubjson: valgrind registers "Invalid write of size 4" in unittests
+#1263: sys: the TLSF implementation contains (a) read-before-write error(s)
+#9: Manual calibration of the CC1100
+
+
+Fixed Issues from the last release (2018.10)
+============================================
+
+#10875: gnrc_ipv6: crash on heavy network load on native
+#10827: boards/nrf51dk: multiple external interrupts not working
+#10768: Build is failing with latest RIOT on windows
+#10761: iotlab-m3: poor ping6 performance and a possibly broken 6lo fragmentation?
+#10753: nanocoap: options buffer overflow
+#10739: sock_dns: Security issues (including remote code execution)
+#10701: gnrc: packet space in gnrc packet buffer is not released on very heavy load
+#10672: gnrc: runs full on very heavy load
+#10634: pyterm: weird behaviour with long lines.
+#10628: Can't read data from BH1750 senseor using STM32F3Discovery
+#10614: DEBUG stops shell if used in i2c init
+#10611: Board z1: BTN0_IN is not defined
```

Comment:
Fixed AFAIK

Reply:
Yes, that's why it's in the closed issues section ;)